src/gc/pal/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+add_compile_options(-fPIC)
+include_directories("../env")
+
+check_include_files(sys/time.h HAVE_SYS_TIME_H)
+check_include_files(sys/mman.h HAVE_SYS_MMAN_H)
```

Comment:
Could you please move these checks into a separate file named configure.cmake to be in line with how we do that for PAL and System.Globalization.Native.so?